src/mailchimp/mailchimp_manager.py

```diff
@@ -107,8 +150,40 @@ def upload_certificates(self, attendees: AttendeeManager,
         Raises:
             TODO: Document potential errors.
         """
-        raise NotImplementedError
-
+        #mailchimp.set_config(self.keys) again shouldnt need to set configurations as the set authorisation function handles that
+        operations = []
+        for attendee in attendees:
+            pdf_file = attendee.file_path
+            with open(pdf_file, "rb") as pdf_file:
+                base64_file = base64.b64encode(pdf_file.read())
+            operation = {
+                "method": "POST",
+                "path": f"/file-manager/files",
+                "operation_id": str(attendee.email),  #if there is an attribute to the attendee object for an id or else a name
+                "body":json.dumps({
+                    "name"  : os.path.basename(pdf_file), #returns a string with the base name of the file path
+                    "file_data": base64_file,
+                    "folder_id": folder_id})  
+                    }
```

Comment:
This ``with`` block needs a ``try`` and ``except`` blocks in case a file IO (``OSError``) occurs. If it does, call ``status_func`` as

``status_func("Pending", 0, num_failed)``